Contribute/get-started-setup-local.md

```diff
@@ -1,150 +1,217 @@
+   :::image type="content" source="media/contribute-get-started-setup-local/fork-location.png" alt-text="Select Fork at the upper right of the repository on GitHub.":::
 
-1. Go to the main repository's GitHub page and click the **Fork** button on the upper right.
+2. You will be taken to a page for creating a new fork. The fork destination should be automatically set to your GitHub account. If you belong to multiple organizations you will be able to change the destination account, choose the account that fits best. Generally the default settings work fine, but feel free to customize the provided fields. Once you are satisfied with your choices, select **Create fork**.
 
-   ![GitHub profile example](./media/contribute-get-started-setup-local/fork.png)
+   :::image type="content" source="media/contribute-get-started-setup-local/create-fork.png" alt-text="Create a new fork page.":::
 
-2. If you are prompted, select your GitHub account tile as the destination where the fork should be created. This prompt creates a copy of the repository within your GitHub account, known as a fork.
+   A copy of the Microsoft repository will be created within the account you selected, this will be your fork.
 
-## Choose a local folder
-Make a local folder to hold a copy of the repository locally. Some of the repositories can be large; up to 5 GB for azure-docs for example. Choose a location with available disk space.
+## Choose a location on your device to clone your fork
 
-1. Choose a folder name should be easy for you to remember and type. For example, consider a root folder `C:\docs\` or make a folder in your user profile directory `~/Documents/docs/`
+Now that you have created a fork of the Microsoft repository, you must clone your fork locally. Some of the documentation repositories can be large; for example, `azure-docs` is up to 25 GB. Make sure to choose a location with available disk space for cloning the forked repository.
+
+When cloning a repository, Git will automatically create a new directory, with the forked repository's name,  within your current directory. If you are working on multiple documentation repositories it might be helpful to store them together. For example, you might want to clone the `azure-docs`, `sql-docs`, and `visualstudio-docs` using a directory path such as `C:/Users/<Username>/Documents/Docs/<Repo-Name>`. With each repository being stored separately in its own directory (`<Repo-Name>`), but together under the same parent directory (`Docs`).
+
+> [!TIP]
+> [Windows Terminal](/windows/terminal/) with [PowerShell](/powershell/) supports a nearly identical command experience to that of Git Bash. You can even run Git Bash inside Windows Terminal.
+
+1. Launch Git Bash
+
+   :::image type="content" source="media/contribute-get-started-setup-local/gitbash-start.png" alt-text="Launch Git Bash from Windows search.":::
```

Comment:
I think this image will be too large. Scale it to around 600px wide for better results.

Reply:
Hmmm, if we're [making this section's commands platform-agnostic](https://github.com/MicrosoftDocs/Contribute/pull/629#discussion_r894952670) maybe remove this image altogether? It's an image of Windows 11 opening Git Bash. Or I could maybe crop it so it isn't so tall. It is a little large, but let me know what you think. This is what it looks like now:

![image](https://user-images.githubusercontent.com/10838153/177758395-8359d1a3-2326-463a-8d4f-dc488eeee1e6.png)